fiware-region-sanity-tests/commons/dbus_phonehome_service.py

```diff
@@ -73,16 +74,22 @@ def timeout(mainloop, logger, *args):
     def phonehome_signal_handler(phonehome_http_data):
```

Comment:
Why don't you implement a new handler for a new signal? I think you are not waiting for a "PhoneHome" call but other type of operation. This way, when emitting this new signal, the rest of listener (other type of tests) waiting for a "PhoneHome" call will not be notified...